packages/TesterApp/scripts/release.js

```diff
@@ -0,0 +1,35 @@
+const { execSync } = require('child_process');
+
+function runCommand(command) {
+  try {
+    execSync(command, { stdio: 'inherit' });
+  } catch (error) {
+    console.error(`Error executing command: ${command}`, error);
+    process.exit(1);
+  }
+}
+
+function buildIOS() {
+  runCommand('rm -rf build/generated');
```

Comment:
uhm, let's ensure the dir before we run `rm -rf`, or maybe force is not necessary here?

Reply:
btw this can be moved outside of the function to the startup code down below

Reply:
Can we use `fs.rm` instead of invoking a shell command?